### PR TITLE
Rewrite coffeescript files because JS on page stopped loading

### PR DESCRIPTION
**Pull request by davidwessman**
After upgrading to Ruby 2.3.0 some JS stopped working.
Removed some unused javascript and rewrote from coffee to JS.

### CODE REVIEW

File: app/assets/javascripts/rents.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+var carCalendar;
+
+carCalendar = function() {
```
Comment (jforberg) [inline app/assets/javascripts/rents.js]: Better written as just ```function carCalendar() {```, var is implied here.

File: app/assets/javascripts/rents.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      });
+    },
+    eventMouseout: function(calEvent, jsEvent) {
+      $(this).css('z-index', 8);
+      return $('.tooltipevent').remove();
+    }
+  });
+};
+
+$('document').ready(carCalendar);
```
Comment (jforberg) [inline app/assets/javascripts/rents.js]: Better written as just ```$(carCalendar);```. both document and ready are implied.

File: app/assets/javascripts/rents.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    eventMouseover: function(calEvent, jsEvent) {
+      var tooltip;
+      tooltip = '<div class="tooltipevent" style="width:auto;height:auto;background:#eb7125;position:absolute;z-index:10001;padding:10px;paddin-top:0px;border:2px solid;">' + calEvent.title + '<br>' + calEvent.start.format('LLL') + '<br>till<br>' + calEvent.end.format('LLL') + '</div>';
+      $('body').append(tooltip);
+      return $(this).mouseover(function(e) {
+        $(this).css('z-index', 10000);
+        $('.tooltipevent').fadeIn('500');
+        return $('.tooltipevent').fadeTo('10', 1.9);
+      }).mousemove(function(e) {
+        $('.tooltipevent').css('top', e.pageY + 10);
```
Comment (jforberg) [inline app/assets/javascripts/rents.js]: Css calls can be chained $(...).css(...).css(...) or  written with object syntax $(...).css({ attr: 1, bttr: 2 }).

File: app/assets/javascripts/rents.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      center: 'title',
+      right: 'month,agendaWeek,agendaDay'
+    },
+    aspectRatio: 1.3,
+    weekNumberTitle: "V ",
+    allDaySlot: false,
+    width: "auto",
+    eventMouseover: function(calEvent, jsEvent) {
+      var tooltip;
+      tooltip = '<div class="tooltipevent" style="width:auto;height:auto;background:#eb7125;position:absolute;z-index:10001;padding:10px;paddin-top:0px;border:2px solid;">' + calEvent.title + '<br>' + calEvent.start.format('LLL') + '<br>till<br>' + calEvent.end.format('LLL') + '</div>';
```
Comment (jforberg) [inline app/assets/javascripts/rents.js]: Could this style be moved to a stylesheet?

**Comment by jforberg** [comment]
Some comments

**Comment by davidwessman** [comment]
Yeah, I need to learn some JS...
Gonna try to clean it up.

**Pull request merged**